src/main/java/gregtech/common/blocks/MetaBlocks.java

```diff
@@ -179,6 +178,11 @@ public static void init() {
                 blockFrame.setRegistryName("frame_" + material.toString());
                 FRAMES.put((SolidMaterial) material, blockFrame);
             }
+            if (material instanceof SolidMaterial && material.hasFlag(GENERATE_METAL_CASING)) {
```

Comment:
do you really need casings from gems?

Reply:
It's a copy from Frames, not a good Idea ?

Reply:
No, bad idea. If you blindly copy stuff, are you sure you want to submit PRs?

Reply:
it's changed to IngotMaterial